ivy/array/elementwise.py

```diff
@@ -1411,23 +1411,39 @@ def round(self: ivy.Array, *, out: Optional[ivy.Array] = None) -> ivy.Array:
         self
             input array. Should have a numeric data type.
         out
-            optional output, for writing the result to. It must have a shape that the
-            inputs broadcast to.
+            optional output array, for writing the result to. It must have a shape that
+            the inputs broadcast to.
 
         Returns
         -------
         ret
-            an array containing the rounded result for each element in ``self``.
-            The returned array must have the same data type as ``self``.
+            an array containing the rounded result for each element in ``self``. The
+            returned array must have the same data type as ``self``.
 
         Examples
         --------
-        With :code:`ivy.Array` input:
+        Using :code:`ivy.Array` instance method:
 
-        >>> x = ivy.array([1.2, 2.4, 3.6])
+        >>> x = ivy.array([6.3, -8.1, 0.5, -4.2, 6.8])
+        >>> y = x.round()
+        >>> print(y)
+        ivy.array([ 6., -8.,  0., -4.,  7.])
+
+        >>> x = ivy.array([-94.2, 256.0, 0.0001, -5.5, 36.6])
+        >>> y = x.round()
+        >>> print(y)
+        ivy.array([-94., 256., 0., -6., 37.])
+
+        >>> x = ivy.array([0.23, 3., -1.2])
+        >>> y = ivy.zeros(3)
+        >>> print(x.round(out=y))
```

Comment:
Since `y` is passed here to the `out` argument, it should be `print(y)` after `x.round(out=y)`